### PR TITLE
Psg 4897 use repo templates

### PR DESCRIPTION
**Pull request by jenn-macfarlane**
## What's New?

Updates README, LICENCE, and package.json files.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have manually tested my code thoroughly
- [x] I have added/updated inline documentation for public facing interfaces if relevant
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing integration and unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by jenn-macfarlane** [comment]
Let me know if I misunderstood what to do here - our [README template example happened to be for passage-node](https://github.com/passageidentity/technical-docs/blob/main/resources/readme-template.md) so I copied that.

Also looks like the PR template used is not coming from this repo/I checked to ensure we don't have any of the PR/Issue/Code of Conduct/Security files we will be inheriting from the Passage org's default community health files. Please let me know if I'm right there too!

File: README.md [hunk 1]
```diff
@@ -1,269 +1,81 @@
...
-let passage = new Passage(passageConfig);
-
-let magicLink = passage.createMagicLink({
-    email: 'newEmail@domain.com',
-    channel: 'email',
-});
-```
+<p align="center">
+    <sub>Passage is a product by <a href="https://1password.com/product/passage">1Password</a>, the global leader in access management solutions with nearly 150k business customers.</sub><br />
+    <sub>This project is licensed under the MIT license. See the <a href="LICENSE.md">LICENSE</a> file for more info.</sub>
```
Comment (agebhardt) [inline README.md:80]: This should be just `LICENSE` to make the relative link work within this repo (sorry I didn't catch this in the template!)

**Review by agebhardt — CHANGES_REQUESTED** [review]
There's just one fix to be made for the license link in the README but otherwise this looks great!

- I confirmed the the docs available on docs.passage.id cover all the code snippets currently in the README, so removing those in favor of the more minimal template is a good choice
- License, Code of Conduct, and Security tabs appear as expected
- I could not confirm that the issue or PR templates were properly being used, but we can address that after this PR is merged.

File: README.md (hunk 1 above)
Comment (jenn-macfarlane) [reply README.md]: Oops, I had fixed that but forgot I updated the branch name. Done!

**Comment by jenn-macfarlane** [comment]
@agebhardt fixed!

Glad you agree with making the technical instructions in this README minimal (taken straight from your example!). My understanding is we want to get as close to this minimal install/import/initialize format in the technical section to minimize drift/duplication of docs. Is my understanding correct?

**Comment by flanagankp** [comment]
I think as part of this work we also want to make corresponding updates to the package.json - namely the description, homepage, and keywords to align with the description, url, and github tags

**Comment by jenn-macfarlane** [comment]
@flanagankp good catch, done. Quick question - there's no risk updating the `homepage`, right? I vaguely recall setting this field in `create-react-app` would change where the project is served from, that's a super specific and unrelated case but has me wondering if we need to be careful about this with any other build systems/stacks.

**Review by agebhardt — APPROVED** [review]

**Pull request merged**